GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -123,6 +123,43 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
             }
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("About Quick Deploy", EditorStyles.boldLabel);
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
```

Comment:
You have a series of horizontal blocks on each section that only contains a single LabelField. If you remove these BeginHorizontal/EndHorizontal does anything change?

Reply:
@thanwah  This is what you'd get:

<img width="718" alt="screen shot 2018-08-28 at 12 42 32 pm" src="https://user-images.githubusercontent.com/18396695/44746722-0179c380-aac0-11e8-97ca-76690573fafb.png">